src/test/java/bio/terra/service/profile/ProfileAPIControllerTest.java

```diff
@@ -141,44 +150,66 @@ public void testUpdateProfileForbidden() {
     verify(profileService, times(0)).updateProfile(eq(billingProfileUpdateModel), eq(user));
   }
 
-  @Test
-  public void testDeleteProfile() {
+  @ParameterizedTest
+  @MethodSource
+  void testDeleteProfile(boolean deleteCloudResources, int expectedAdminAuthNumberOfInvocations) {
     when(authenticatedUserRequestFactory.from(any())).thenReturn(user);
     UUID deleteId = UUID.fromString("aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee");
     String jobId = "jobId";
-    when(profileService.deleteProfile(eq(deleteId), eq(user))).thenReturn(jobId);
+    when(profileService.deleteProfile(eq(deleteId), eq(deleteCloudResources), eq(user)))
+        .thenReturn(jobId);
```

Comment:
if all parameters to a mock are constant, you don't need to use `eq()`.
```suggestion
    when(profileService.deleteProfile(deleteId, deleteCloudResources, user)).thenReturn(jobId);
```

Reply:
Ah, ok - You don't need it when mocking but do on verify.